.ci/generate-releases.sh

```diff
@@ -172,7 +175,7 @@ cd releases
   echo "next_patch_8=$next_patch_8"
   echo "edge_8=$edge_8"
   echo "generated=https://github.com/elastic/apm-pipeline-library/actions/workflows/generate-elastic-stack-releases.yml"
-} > releases.properties
+} | tee releases.properties
```

Comment:
print something in the console while this runs so we can see what versions will be shown.